package.json

```diff
@@ -18,7 +18,7 @@
   "devDependencies": {
     "@typescript-eslint/parser": "^2.4.0",
     "babel-eslint": "^10.0.3",
-    "eslint": ">=2.0.0",
+    "eslint": ">=2.0.0 <7.0.0",
```

Comment:
This is because I got a warning when installing the dependencies (`@typescript-eslint/parser@2.34.0 requires a peer of eslint@^5.0.0 || ^6.0.0 but version 7.4.0 was installed.`), so I had to limit the eslint version to prevent the tests from failing.